src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -217,6 +217,12 @@ public boolean isExternalSubsParsed() {
 	private byte[] h264_annexB;
 
 	/**
+	 * @deprecated Use standard getter and setter to access this variable.
+	 */
+	@Deprecated
+	public String truncated;
+
+	/**
 	 * Not stored in database.
```

Comment:
@Sami32 I think "package private" is mysterious for everyone but those who designed Java 👼 

> Just because i become suspicious, you don't work on that, right ?! Because i didn't asked for, and i don't think it's should worst of it.

I'm not sure exactly what you mean here: Are you asking if I'm working on this branch? If so, I'm not, but I interpreted your comment further up as if you wanted it reviewed:

> Something like it's ready for me.

I was starting to look at it for review when I saw that public/private thing. Some of the changes I didn't understand, I'm unsure of what the goal are. I didn't get any further on my review as it took some time to explain the private/public concept, and I had to work on my car.

When it comes to plugins there's really no way to know what plugins there are. There are some plugins that "we know of", but there are other plugins out there. I saw one plugin that implemented the GUI settings as a web interface. Which plugins are updated and in use is also impossible to know. In short, it's impossible to be sure of anything when it comes to plugins.

I suspect that there are only a few that's actually in use though, and since neither of them has been updated for the later versions of UMS the number of users can't be that many or we would probably have heard it.